src/_actions/SettingsActions.js

```diff
@@ -19,3 +19,8 @@ export const serverDataAccountSettings = serverResponse => ({
     type: types.SERVER_DATA_ACCOUNT_SETTINGS,
     serverResponse,
 });
+
+export const updatePersonalAddress = newAddress => ({
```

Comment:
I had a more generic action elsewhere.
I suggest it for settings too:
LIke UPDATE_SETTINGS_FIELD, it will have a fieldName/fieldKey and fieldValue params, so you use the same action for all settings.